controllers/factory/vmalert.go

```diff
@@ -357,7 +357,7 @@ func vmAlertSpecGen(cr *victoriametricsv1beta1.VMAlert, c *config.BaseOperatorCo
 		confReloadArgs = append(confReloadArgs, fmt.Sprintf("-volume-dir=%s", path.Join(vmAlertConfigDir, cm)))
 	}
 	for arg, value := range cr.Spec.ExtraArgs {
-		args = append(args, fmt.Sprintf("--%s=%s", arg, value))
+		args = append(args, fmt.Sprintf("-%s=%s", arg, value))
```

Comment:
Can you update `vmagent` and `vmsingle` as well for consistency?

Reply:
Updated